cranelift/isle/veri/veri_engine/src/annotations.rs

```diff
@@ -259,6 +259,19 @@ fn spec_op_to_expr(s: &SpecOp, args: &Vec<SpecExpr>, pos: &Pos, env: &ParsingEnv
                 .collect();
             Expr::Switch(Box::new(swith_on), arms)
         }
+        SpecOp::Load => {
+            assert_eq!(
+                args.len(),
+                3,
+                "Unexpected number of args for subs operator {:?}",
```

Comment:
Typo fix: This error message should say `load`, not `subs`

Reply:
We changed the message to include load instead of subs.